docker-compose/config/scheduler.jsonnet

```diff
@@ -14,4 +14,16 @@ local common = import 'common.libsonnet';
   browserUrl: common.browserUrl,
   contentAddressableStorage: common.blobstore.contentAddressableStorage,
   maximumMessageSizeBytes: common.maximumMessageSizeBytes,
+  executeAuthorizer: { allow: {} },
+  actionRouter: {
+    simple: {
+      platformKeyExtractor: { actionAndCommand: {} },
+      invocationKeyExtractors: [{ requestMetadata: {} }],
+      initialSizeClassAnalyzer: {
+        defaultExecutionTimeout: '1800s',
+        maximumExecutionTimeout: '7200s',
+      },
+    },
+  },
+  platform_queue_with_no_workers_timeout: '900s',
```

Comment:
Also make sure to camelCase this field, for consistency with the rest of the config.